pkg/manifest/anaconda_installer_iso_tree.go

```diff
@@ -560,3 +556,28 @@ func makeISORootPath(p string) string {
 	fullpath := path.Join("/run/install/repo", p)
 	return fmt.Sprintf("file://%s", fullpath)
 }
+
+func makeKickstartSudoersPost(names []string) string {
+	echoLineFmt := `echo -e "%[1]s\tALL=(ALL)\tNOPASSWD: ALL" > "/etc/sudoers.d/%[1]s"
+chmod 0440 /etc/sudoers.d/%[1]s`
+
+	filenames := make(map[string]bool)
```

Comment:
(sooooory super nitpick) this is totally fine as it is, while reading it I was wondering:
```
sort.Strings(names)
names = clies.Compact(names)
```
might be a simpler way to remove duplicates from the inputs than the map but I guess it's really fine either way :)

Reply:
If we weren't already looping through the elements, this would be nice, but since we are, it feels simpler to just keep track.  But I don't mind either way.

Reply:
totally fine